app/components/object-detail-widget/template.hbs

```diff
@@ -41,7 +41,12 @@
         <div>
             <h3>Identifiers</h3>
             <ul>
-                {{#each objectData._source.identifiers as |identifier|}}
+                {{#each identifiers.links as |identifier|}}
```

Comment:
> Is there an order for links? This would lose any order that SHARE has on that field

@laurenbarker is there any significance to the order of the `identifiers` field?